### ISSUE TITLE
Where is the actual .NET wrapper project?

### ISSUE THREAD

**Issue by sindizzy** [Esri/file-geodatabase-api#18]
If I want to tweak the .NET wrapper where can I find that project?

**Comment by donnyv** [comment]
ESRI hasn't open sourced any code. They just let you use the dlls for free. They call that "open".

**Comment by sindizzy** [comment]
Yes I get that. That's why Im asking for the .net wrapper. That code just wraps the DLLs.

**Comment by donnyv** [comment]
Yeah thats included in the "ESRI is not gonna give you any code" pledge. ;-)
